great-idea/css/index.css

```diff
@@ -58,9 +58,79 @@ html, body {
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
+    font-family: 'Merriweather', serif;
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+section, .fullRow{
+	margin-top: 30px;
+	margin-bottom: 30px;
+}
+
+.container{
+	width: 80%;
+	margin: 0 auto;
+}
+
+.menuList, .topBar, .catSection, .leftCat, .fullRow{
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.topBar, .catSection, .fullRow{
+	justify-content: space-evenly;
+}
+
+.leftCat{
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	text-align: center
+}
+
```

Comment:
Using ```.catSection {
  flex-wrap: nowrap;
}```   override wrap with nowrap it will force the children to stay on the same row , or you can put `width` to both  children , we can talk more tomorrow about this